backend/services/table_anomaly_service.py

```
@@ -436,16 +413,248 @@ async def _detect_duplicates(self, conn, table_name: str) -> int:
+                    ORDER BY duplicate_count DESC
+                    LIMIT 5
+                """).format(
+                    sql.Identifier("public"),
+                    sql.Identifier(table_name)
+                ))
+                duplicate_data = cursor.fetchall()
+                
+                if duplicate_data:
+                    column_names = [desc[0] for desc in cursor.description]
+                    for row in duplicate_data:
+                        record = dict(zip(column_names, row))
```

Ruff (0.13.1)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)